pkg/nimble/scanner/include/nimble_scanner.h

```diff
@@ -60,8 +52,7 @@ typedef void(*nimble_scanner_cb)(uint8_t type,
  *                      default parameters
  * @param[in] disc_cb   callback triggered of each received advertising packet
  *
- * @return  NIMBLE_SCANNER_OK on success
- * @return  NIMBLE_SCANNER_ERR if putting NimBLE into discovery mode failed
+ * @return  0 on success
```

Comment:
No error value anymore? Then why have a return value at all?

Reply:
For API stability. If something changes in the initialization at some point, the API does not break if an error return value is added then...